DigitalExcellence/dex-backend
# Feature/272 max file size allowed extensions annotations

## Description

Added action annotations to flexibly change allowed file size and annotations.
These can be added above actions and controllers themselfs.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Upload a file that is not a png, jpg, jpeg or gif that is below 2mb.
2. Upload a file that is more than 2mb.
3. These should both return a badrequest.

## Link to issue

Closes: [#272](https://github.com/DigitalExcellence/dex-backend/issues/272)
